modules/custom/d_p_banner/templates/paragraph--d-p-banner.html.twig

```diff
@@ -54,7 +58,7 @@
         } only %}
       {% endblock %}
       <div class="content-wrapper">
-        <div class="content container">
+        <div class="content content-inside-wrapper container">
```

Comment:
tutaj te modyfikacje są dlatego, żeby poprawnie dodawały się paddingi, w przypadku włączenia flagi half-transparent (która właśnie sprawia, że ten właściwy content jest jakby przeniesiony do osobnego wrappera i modyfikatory powinny wpływać tylko na ten wrapper - jedynie czego nie zmieniłem, to modyfikator paragraph-theme nie wpływa na tę półprzezroczystą połówkę.. trochę to było problematyczne, cięzko było zdecydować co miałoby się dziać z kolorem tego overlaya na obrazku... Dlatego, bez custom-theme bez wybranej flagi half-transparent jest stosowany w 100%:

![image](https://user-images.githubusercontent.com/37533838/102213191-aa899980-3ed6-11eb-8680-232a31ed071a.png)


a jezeli user wybierze half-transparent, to bedzie tak:
![image](https://user-images.githubusercontent.com/37533838/102213261-c68d3b00-3ed6-11eb-804f-f4730831ba56.png)